david_cpp.cc

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+using namespace std;
+
+int main(){
+	cout<<"Hello Martina!";
+	cout<<"\n I can do this.";
```

Comment:
Please have a look at .... so that this can be done easier. 
Please name your variables in a more readable way.

Reply:
But I do not have any variables.